presto-native-execution/presto_cpp/presto_protocol/connector/arrow_flight/presto_protocol_arrow_flight.h

```diff
@@ -38,15 +38,38 @@ void to_json(json& j, const ArrowTransactionHandle& p);
 void from_json(const json& j, ArrowTransactionHandle& p);
 
 } // namespace facebook::presto::protocol::arrow_flight
+/*
```

Comment:
@BryanCutler Why is this copyright message inserted here again ? Please can you fix it if possible.

Reply:
The special file `ArrowColumnHandle.hpp.inc` has a copyright header and when the protocol is generated it puts the entire file into `presto_protocol_arrow_flight.h`. It's the same thing for the Hive connector and all the *.inc files.
I thought it was a bit strange too, but it seems like the *.inc files need a header, so it's just the side effect.

Reply:
We should fix this in the scripting. Please create an issue for it.